locations/spiders/tower_loan_us.py

```diff
@@ -5,12 +5,17 @@
 from scrapy.linkextractors import LinkExtractor
 from scrapy.spiders import CrawlSpider, Rule
 
+from locations.categories import Categories
 from locations.dict_parser import DictParser
 
 
 class TowerLoanUSSpider(CrawlSpider):
     name = "tower_loan_us"
-    item_attributes = {"brand": "Tower Loan", "brand_wikidata": "Q126195732"}
+    item_attributes = {
+        "brand": "Tower Loan",
+        "brand_wikidata": "Q126195732",
+        "extras": Categories.OFFICE_FINANCIAL.value,
```

Comment:
This should match NSI https://github.com/osmlab/name-suggestion-index/blob/233a917b0ebab177d438b68721180a93eeb45465/data/brands/shop/money_lender.json#L415

Reply:
Yeah it won't match until NSI is next released, usually a week or 2.